theme/templates/resource-landing-page/top-right-buttons.html

```diff
@@ -68,7 +68,7 @@
                 {% endif %}
                 {% if user.is_authenticated and rights.statement %}
```

Comment:
@pkdash I initially thought about that, but then discovered I actually need to return three states (true, false, N/A) rather than 2 states (true, false) since for example, if user is not authenticated, this button does not show up at all rather than showing as disabled. I also want to reuse the routine I put in for both template check and for internal function permission check which would be called by REST API in the future. Since the same function is used by both copy and new version actions which have different enabling permission checks, it would involve more code and be potentially more confusing if I check the whole permission out of templates. Since the current logic in the template is not complicated, I think it is clearer to do it this way. I'd suggest to leave it as is for now given the reasons I give above.